test/new-e2e/tests/agent-platform/common/bound-port/windows.go

```diff
@@ -0,0 +1,45 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package boundport
+
+import (
+	"encoding/json"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client"
+)
+
+func boundPortsWindows(client client.VM) ([]BoundPort, error) {
```

Comment:
Would be nice to have something similar to what you did for `process/` to define the windows logic in the `windows` test side.  I guess you did not do it because of circular dependency it would create

Reply:
I started to but wasn't sure where to put the `BoundPort` type. Do you have a suggestion?

Reply:
oh, I always forget that Go interfaces don't need to be referenced directly. updated.